wgsl/index.bs

```diff
@@ -4371,8 +4433,8 @@ For details on conversion to and from floating point types, see [[#floating-poin
   <tr algorithm="scalar reinterpretation from unsigned to signed">
       <td>|e|: u32<td>`i32(`|e|`)`: i32
       <td>Reinterpretation of bits.<br>
-          The result is the unique value in [=i32=] that is equal to (|e| mod 2<sup>32</sup>).
-  <tr algorithm="scalar conversion from binary32 floating point to signed integer">
+          The result is the unique value in [=i32=] that has the same bit pattern as |e|.
```

Comment:
Missing AbstractInt here?

Reply:
Ah, discussed offline:  The AbstractInt -> i32 conversion is already the default. So  writing `i32( ... ) ` around an AbstractInt already just works as intended.

Cool